<launch>
  <arg name="connection_string" default="HostName=sample.azure-devices.net;DeviceId=rosbot;x509=true" />
  <arg name="authentication_type" default="x509" />
  <arg name="private_key_file" default="$(env DEVICE_PRIVATE_KEY)" />
  <arg name="public_key_file" default="$(env DEVICE_PUBLIC_KEY)" /> 
  <node pkg="roscpp_azure_iothub" type="roscpp_azure_iothub_node" name="roscpp_azure_iothub_node" output="screen">
    <rosparam subst_value="true">
      connection_string: $(arg connection_string)
      authentication_type: $(arg authentication_type)
      private_key_file: $(arg private_key_file)
      public_key_file: $(arg public_key_file)
    </rosparam>
  </node>
</launch>
